rocAL/source/pipeline/master_graph.cpp

```diff
@@ -885,6 +934,16 @@ MasterGraph::get_output_tensors() {
     return &_output_tensor_list;
 }
 
+bool MasterGraph::is_out_of_data() {
+    // If any of the loader module's remaining count is less than the batch size, return loader out of data
+    for (auto& loader_module : _loader_modules) {
+        if (loader_module->remaining_count() < (_is_sequence_reader_output ? _sequence_batch_size : _user_batch_size)) {
```

Comment:
this is ambiguous. Did you mean rem_count < _user_batch_size?

Reply:
Yes Rajy, In this function we need to get the least remaining count of all the loaders
We check if remaining count is less than the user batch size, special case for sequence reader also handled here